src/app/lists/loading.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+
+import { CircularProgress } from "@mui/material";
+import React from "react";
+import { styled } from "styled-components";
+
+const Centerer = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 80vh;
+`;
+
+export const Loading: React.FC<{}> = () => {
```

Comment:
can this just be the main Loading page? I don't see why we need a separate component for lists when we can just improve the root one :)